build/configs/rtl8721csm/cpp_loadable/defconfig

```diff
@@ -1154,6 +1154,9 @@ CONFIG_LIBC_SYMTAB=y
 #
 CONFIG_C99_BOOL8=y
 CONFIG_HAVE_CXX=y
+CONFIG_CXX_VERSION_11=y
```

Comment:
Why not add the version support for all cxx configs for all other boards ?

Reply:
Done

Reply:
What about ??
../build/configs/imxrt1050-evk/helloxx/defconfig
../build/configs/imxrt1020-evk/helloxx/defconfig

May be we can ignore artik053 Make.defs & defconfigs since it is obsolete. What is your opinuion @sunghan-chang ?

Reply:
defconfig changes are not required as I changed the version check logic as suggested by @sunghan-chang 

```
ifeq ($(CONFIG_CXX_VERSION_17),y)
  CXXFLAGS += -std=c++17
else ifeq ($(CONFIG_CXX_VERSION_14),y)
  CXXFLAGS += -std=c++14
else
  CXXFLAGS += -std=c++11
endif
```

by default, if no config is set we use cpp11.